db/seeds.rb

```diff
@@ -0,0 +1,122 @@
+users = User.create([
+{name: 'Admin',
+ role: 1,
+ email: "admin@gmail.com",
+ password: "123123" },
+])
+
+cars = Car.create([
+{
+name: 'BMW F10',
+image: 'https://www.pngmart.com/files/22/BMW-F10-PNG-HD-Isolated.png',
+model: '2021',
+price: 140,
+description: 'The F10 is the model code BMW uses internally for the sixth-generation 5 Series sedan, produced between 2010 and 2016.'
+},
+{name: 'Renault Cio',
+ image: 'https://www.pngmart.com/files/10/Renault-PNG-Image.png',
+ model: 2018,
+ price: 139,
+ description: 'The 2018 Renault Clio was available in eight variants, is classed as a Light Car and was built in France.' },
+{name: 'Volkswagen Golf',
+ image: 'https://www.pngmart.com/files/22/Volkswagen-Golf-SV-PNG-Isolated-Pic.png',
+ model: '2020',
+ price: 200,
+description: '2020 Golf is a good used car. 
+Its handling and engine power are good enough to provide an entertaining driving experience. ',
+},
+{name: 'Mercedes Viano',
+ image: 'https://stimg.cardekho.com/images/carexteriorimages/630x420/Mercedes-Benz/Mercedes-Benz-Viano/2680/1562819292786/front-left-side-47.jpg',
+ model: Viano,
```

Comment:
- [ ] When I try to seed the db, this happens:

![image](https://user-images.githubusercontent.com/57421823/214324648-533595ee-2434-4a69-8947-bb310d149c7f.png)

## Explanation of the problem

On line 30, `model: Viano`.  `Viano` is not a string (missing quotes because it should be a string).  Therefore, Rails tries to look for a method or variable called `Viano`, and of course, it finds anything with that name.

## Solution

Thus:

```ruby
model: 'Viano'
```

This worked for me.

Kindly, fix this bug so your db works as intended and gets properly seeded.  💪

Reply:
@Meltrust thank you so much for this. 
I updated it!